app/posts/manage-teacher-training-applications/2021-02-25-filter-by-subject.md

```diff
@@ -65,6 +67,7 @@ We considered some changes which were not implemented.
 
 - Reordering the subject list to show all checked subjects in alphabetical order at the top of the list of subjects, followed by an alphabetical list of unchecked subjects. This was not implemented as it is known to increase error rates and misinterpretations.
 - Providing a ‘selected’ count below the filter’s heading similar to that found in the GOV.UK finder frontend component called [option select](https://finder-frontend.herokuapp.com/component-guide/option-select). This was not implemented in this iteration as we could not establish a need for it.
+- Putting each filter section into an expandable region, but this hides content and adds unnecessary interactions.
```

Comment:
Can we be more specific about ‘unnecessary interactions’?

Do you mean ‘hiding the content means the user has to work harder to reveal the content’?

Reply:
I have made the suggested change